src/api/che-api-client.ts

```diff
@@ -325,6 +351,8 @@ export class CheApiClient {
         return new Error(`E_CHE_API_UNAUTHORIZED - Endpoint: ${endpoint} - Message: ${JSON.stringify(error.response.data)}`)
       } else if (status === 404) {
         return new Error(`E_CHE_API_NOTFOUND - Endpoint: ${endpoint} - Message: ${JSON.stringify(error.response.data)}`)
+      } else if (status === 503) {
+        return new Error(`E_CHE_API_UNAVAIL - Endpoint: ${endpoint} returned 503 code`)
```

Comment:
Without including `response.data` ?

Reply:
Because there is no response or a default 503 page.